sdk/helper/tokenhelper/tokenhelper.go

```diff
@@ -38,6 +38,9 @@ type TokenParams struct {
 
 	// The TTL to user for the token
 	TokenTTL time.Duration `json:"token_ttl" mapstructure:"token_ttl"`
+
+	// The maximum number of times a token issued from this role may be used.
+	TokenNumUses int `json:"token_num_uses,omitempty" mapstructure:"token_num_uses,omitempty"`
```

Comment:
I'm not sure if this should be omitempty or not, but I'm pretty sure we should be consistent with the other parameters, whether we make them all omitempty or none.

Reply:
Also note that because `PopulateTokenData` adds to the map regardless, this actually has no effect currently.